src/adm/settings.rs

```diff
@@ -129,6 +129,12 @@ pub struct AdmAdvertiserFilterSettings {
         default
     )]
     pub(crate) click_hosts: Vec<Vec<String>>,
+    #[serde(
+        deserialize_with = "deserialize_hosts",
+        serialize_with = "serialize_hosts",
+        default
+    )]
+    pub(crate) img_hosts: Vec<Vec<String>>,
```

Comment:
nit: not terribly important but might as well match the field name in the json's prefix
```suggestion
    pub(crate) image_hosts: Vec<Vec<String>>,
```